wg-iot-edge/README.md

```diff
@@ -47,5 +47,7 @@ are then managed by the appropriate SIGs based on impacted areas of code.
 * Evaluate and extend existing CLI tools to manage k8s clusters running in remote edge locations.
 
 It is an open and active working group, and we always welcome new additions!
+## Publications
+* [Edge security challenges](whitepapers/edge-securuity-challenges/)
```

Comment:
Can you also correct the main name while we are here? s/securuity/security

Reply:
Right now the blog post from CNCF links to this place. I dont know if we should change it and break the link